store/migration/dev/LATEST.sql

```diff
@@ -170,7 +170,8 @@ CREATE TABLE project (
     db_name_template TEXT NOT NULL,
     role_provider TEXT NOT NULL CHECK (role_provider IN ('BYTEBASE', 'GITLAB_SELF_HOST', 'GITHUB_COM')) DEFAULT 'BYTEBASE',
     schema_version_type TEXT NOT NULL CHECK (schema_version_type IN ('TIMESTAMP', 'SEMANTIC')) DEFAULT 'TIMESTAMP',
-    schema_change_type TEXT NOT NULL CHECK (schema_change_type IN ('DDL', 'SDL')) DEFAULT 'DDL'
+    schema_change_type TEXT NOT NULL CHECK (schema_change_type IN ('DDL', 'SDL')) DEFAULT 'DDL',
+    lgtm_check TEXT NOT NULL CHECK (lgtm_check IN ('DISABLED', 'PROJECT_OWNER', 'PROJECT_MEMBER')) DEFAULT 'DISABLED'
```

Comment:
How about doing a json struct for this file instead of an enum? This gives us opportunity to extend.